pkg/util/utils.go

```diff
@@ -749,3 +773,15 @@ func SizeOfPath(path string) (uint64, error) {
 	})
 	return size, err
 }
+
+func IsPodmanMachine() bool {
```

Comment:
I think we need to move this in c/common,
https://github.com/containers/common/blob/112a47964ddbe816349ea412c0073902e805f943/libnetwork/etchosts/ip.go#L19 needs to know if it is a machine. Assuming we want to remove the config field in the future.
Also just looking at this I guess the condition should also only be set for qemu and not wsl since there is no gvproxy.

Reply:
Lets do this in a separate PR, since I would like to get this in for RCs.
/lgtm